specification/cdn/resource-manager/Microsoft.Cdn/preview/2022-01-01-preview/afdx.json

```diff
@@ -0,0 +1,6282 @@
+          "format": "int64",
+          "description": "The current value of the usage."
+        },
+        "limit": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The limit of usage."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageName",
+          "description": "The name of the type of usage."
+        }
+      },
+      "required": [
+        "unit",
+        "currentValue",
+        "limit",
+        "name"
+      ],
+      "description": "Describes resource usage."
+    },
+    "UsagesListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "The list of resource usages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of results."
+        }
+      },
+      "description": "The list usages operation response."
+    },
+    "ActivatedResourceReference": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "isActive": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Whether the resource is active or inactive"
+        }
+      },
+      "description": "Reference to another resource along with its state.",
+      "type": "object",
+      "x-ms-azure-resource": true
+    },
+    "Policy": {
+      "description": "A policy is a logical grouping of policies",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AFDStateProperties"
+        },
+        {
+          "$ref": "./cdn.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyProperties"
+        }
+      }
+    },
+    "PolicyProperties": {
+      "description": "The JSON object that contains the properties of the Policy to create.",
+      "type": "object",
+      "required": [
+        "policyData",
+        "serviceType",
+        "policyVersion"
+      ],
+      "properties": {
+        "policyData": {
+          "description": "Policy to be applied",
+          "type": "string"
+        },
+        "policyVersion": {
+          "description": "Schema version of the policy",
+          "type": "string"
+        },
+        "serviceType": {
+          "description": "Service type of the policy",
+          "type": "string"
+        }
+      }
+    },
+    "PolicyGetParameters": {
+      "description": "The JSON object that contains the properties of the Policy to update.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AFDStateProperties"
+        },
+        {
+          "$ref": "./cdn.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "policyVersion": {
```

Comment:
just return a "Policy" definition on GET.  This definition is not correct since "policyVersion", etc... is not within an inner "properties" object

Reply:
I fixed this issue

Reply:
Why isn't policyData returned in responses or on GET?  Is it a secret?  This should still not have its own definition and instead if you want this behavior you should return the "Policy" definition and add "x-ms-mutability" to the policyData field.  https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability